biz.aQute.repository/test/SomeTest.java

```diff
@@ -0,0 +1,11 @@
+import org.junit.Test;
+
+public class SomeTest {
```

Comment:
What is the point of this file? It does nothing except add one more passing test.